django/db/migrations/state.py

```diff
@@ -579,11 +580,19 @@ def render(self, apps):
         body.update(self.construct_managers())
 
         # Then, make a Model object (apps.register_model is called in __new__)
-        return type(
-            str(self.name),
-            bases,
-            body,
-        )
+        try:
+            return type(
```

Comment:
It seems these could fit on a single file. We could define `args = (str(self.name), bases, body)` to avoid repeating them.